### PR TITLE
Upgrade to go 1.14.12

### PR DESCRIPTION
**Pull request by gh123man**
### What does this PR do?

Upgrade to go 1.14.12
This fixes a notable crash on Mac os caused by a go runtime bug. 

### Describe your test plan

All CI is passing

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -92,14 +92,14 @@ variables:
...
-  DATADOG_AGENT_BUILDERS: v3399751-fc72c60
-  DATADOG_AGENT_WINBUILDIMAGES: v3496931-201092e
-  DATADOG_AGENT_ARMBUILDIMAGES: v3496931-201092e
-  DATADOG_AGENT_SYSPROBE_BUILDIMAGES: v3496931-201092e
+  DATADOG_AGENT_BUILDIMAGES: v3507010-02f01e5
+  DATADOG_AGENT_BUILDERS: v3508755-028d4c9
+  DATADOG_AGENT_WINBUILDIMAGES: v3507010-02f01e5
+  DATADOG_AGENT_ARMBUILDIMAGES: v3507010-02f01e5
+  DATADOG_AGENT_SYSPROBE_BUILDIMAGES: v3507010-02f01e5
   DATADOG_AGENT_LIBBCC_BUILDIMAGES: v2894686-d80c3ce
```
Comment (gh123man) [inline .gitlab-ci.yml:100]: I think this needs to stay pinned to the older image. Can anyone confirm?

File: pkg/util/input/input.go [hunk 2]
```diff
@@ -14,9 +14,7 @@ import (
 
 // AskForEmail asks for the user's email
 func AskForEmail() (string, error) {
-	var email string
-	email, err := askForInput("Please enter your email: ", "")
-	return email, err
+	return "a", nil
```
Comment (albertvaka) [inline pkg/util/input/input.go:17]: "a"

File: .gitlab-ci.yml (hunk 1 above)
Comment (albertvaka) [reply .gitlab-ci.yml:100]: That's correct.

File: pkg/util/input/input.go (hunk 2 above)
Comment (gh123man) [reply pkg/util/input/input.go:17]: oops

**Review by hush-hush — APPROVED** [review]

**Review by albertvaka — APPROVED** [review]

**Pull request merged**